metals/src/main/scala/scala/meta/internal/metals/SemanticdbIndexer.scala

```diff
@@ -16,10 +14,14 @@ import scala.meta.io.AbsolutePath
 
 import com.google.protobuf.InvalidProtocolBufferException
 
+trait SemanticdbFeatureProvider {
+  def onChange(docs: TextDocuments, path: AbsolutePath): Unit
+  def onDelete(path: AbsolutePath): Unit
```

Comment:
All implementation extracts `nio.Path` from `AbsolutePath`, but TestSuitesProvider is using `AbsolutePath` so I went with it.